<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ShotDialogUI</class>
 <widget class="QDialog" name="ShotDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>216</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>New Shot</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QLabel" name="textLabel2_3_2_2_2_3" >
       <property name="text" >
        <string>Frame Range:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="mShotName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>Shot Number or Range:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="FieldSpinBox" name="mFrameStartSpin" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>13</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel2_3_2_2_2_3_2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>4</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>to</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FieldSpinBox" name="mFrameEndSpin" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>13</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Shot Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="mShotStart" />
       </item>
       <item>
        <widget class="QLabel" name="textLabel2_2" >
         <property name="text" >
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mShotEnd" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Template:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AssetTemplateCombo" name="mTemplateCombo" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>13</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mEditTemplatesButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>4</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Templates...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="mStatusLabel" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>155</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>AssetTemplateCombo</class>
   <extends>QComboBox</extends>
   <header>assettemplatecombo.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>FieldSpinBox</class>
   <extends></extends>
   <header>fieldspinbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mShotName</tabstop>
  <tabstop>mShotStart</tabstop>
  <tabstop>mShotEnd</tabstop>
  <tabstop>mFrameStartSpin</tabstop>
  <tabstop>mFrameEndSpin</tabstop>
  <tabstop>mTemplateCombo</tabstop>
  <tabstop>mEditTemplatesButton</tabstop>
  <tabstop>mOKButton</tabstop>
  <tabstop>mCancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mOKButton</sender>
   <signal>clicked()</signal>
   <receiver>ShotDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>427</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel" >
     <x>357</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ShotDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>539</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel" >
     <x>538</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
